testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/OIDCJwksClientRegistrationTest.java

```diff
@@ -106,7 +104,7 @@ public void createClientWithJWKS_generatedKid() throws Exception {
 
         // Generate keys for client
         TestOIDCEndpointsApplicationResource oidcClientEndpointsResource = testingClient.testApp().oidcClientEndpoints();
-        Map<String, String> generatedKeys = oidcClientEndpointsResource.generateKeys();
+        Map<String, String> generatedKeys = oidcClientEndpointsResource.generateKeys(null);
```

Comment:
Add oidcClientEndpointsResource.generateKeys() rather than using null here.

Reply:
Instead, is it OK to change null onto "RS256" to specify the algorithm explicitly?

Reply:
That works